data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+	to offer
+		has "<uhai> 0: Bring Hai: done"
+	substitutions
+		<prioritymessage> `Unfortunately, because it's in Hai, you can only guess at the meaning of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds.`
+			not "language: Hai"
+		<prioritymessage> `"Assistance requested from all available pilots, Unfettered or not. Local outbreak of broiling sickness has spread catastrophically in our space, and is now classified as an uncontrolled [epidemic, plague]. Hundreds of Hai are already critically ill and Unfettered hospitals have declared an emergency situation, fearing the consequences of current [medical, treatment] supply levels. Medical supplies are to be retrieved from Stonebreak for immediate return."`
+			has "language: Hai"
+	on offer
+		conversation
+			`There is a message repeating on priority channels in Hai standard, tagged with the information for one of the hospitals nearby. <prioritymessage> Unsure whether they would accept your assistance as an outsider, especially given the message is in Hai, you decide to first look for more information planet-side. After all, it is unlikely that a Hai disease would be of any risk to you.`
+			`	When landing on an Unfettered planet, even after having brought them a jump drive, there is always some lingering tension. Except that now you feel that tension more than ever before, probably because of the medical emergency. Unfettered rush around, and various vehicles are frequently coming and going, bringing both supplies and the infected to the hospital. The hospital itself looks different than the rest of the Unfettered structures: it seems more organized, proper. It is probably amongst the best maintained ones here, alongside the shipyards, judging from the outside.`
+			`	As you get closer you notice that one part of it appears as though it had been grafted onto the rest, and recently at that. Not even their most critical buildings appear to be totally safe from the Unfettered's propensity for haphazard building extensions.`
+			`	Many pilots have taken up the call, but it appears the local medical workers are still interested in all the volunteers they can get. Now that you're here, you're not sure you'll be able to leave without at least being accosted for your assistance with this.`
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+					to display
+						has "language: Hai"
+					goto agree
+				`	(Ask a local ship to translate.)`
+					to display
+						not "language: Hai"
+					goto translation
+				`	(I don't have time for this.)`
+			label leave
+			`	You manage to leave without anyone stopping you in all the rush, though you're sure someone was trying to get your attention. If they remember you, the Unfettered will likely not be too happy. You decide to run off immediately in case someone noticed. The Chief that asked for your assistance will probably be disappointed about this.`
```

Comment:
```suggestion
			`	You manage to leave without anyone stopping you in all the rush, though you're sure someone was trying to get your attention. If they remember you, the Unfettered will likely not be too happy. You decide to take off immediately in case someone noticed. The Chief that asked for your assistance will probably be disappointed about this.`
```